com/win32com/src/extensions/PyGEnumVariant.cpp

```diff
@@ -63,7 +63,7 @@ STDMETHODIMP PyGEnumVARIANT::Reset(void)
 }
 
 STDMETHODIMP PyGEnumVARIANT::Clone(
-    /* [out] */ IEnumVARIANT __RPC_FAR *__RPC_FAR *ppEnum)
+    /* [out] */ IEnumVARIANT __RPC_FAR * __RPC_FAR *ppEnum)
```

Comment:
please remove these types of changes.

Reply:
I see these came from clang-format? I guess I wouldn't necessarily object to a single commit which had just clang formatting done with a git-blame-ignore entry too if necessary - but our CI already runs clang-format, so I'm a little confused why this is coming up now?